Framework/PythonInterface/test/python/plugins/algorithms/IndexPeaksTest.py

```diff
@@ -19,22 +19,13 @@ class IndexPeaksTest(unittest.TestCase):
     def test_exec_with_different_args(self):
         # load data
         Load(
-            Filename="test_index_satellite_peaks.integrate", OutputWorkspace="test",
+            Filename="test_index_peaks.integrate", OutputWorkspace="test",
         )
         FindUBUsingIndexedPeaks(PeaksWorkspace="test")
 
         # defualt args
         IndexPeaks(PeaksWorkspace="test", Tolerance=0.12)
 
-        # testing zero (default) modVec with maxOrder=1
-        IndexPeaks(
```

Comment:
Is it worth checking if we can get these tests working? Maybe as a future issue if that's difficult - it would still be good to get the rest of these tests in for now seeing as none were being run till now.

Reply:
@RichardWaiteSTFC This fails now as something in the past has changed, do you know if this algorithm call need tweaking / is still valid?

Reply:
The call to IndexPeaks looks OK to me - don't think anything has changed recently. The last thing I can find was one of mine from August (just changing a warning).
https://github.com/mantidproject/mantid/commit/11886ae3a86c923246d3222829358702e05b3a86#diff-512d369be72489a814cd4017a6c21a497033e2e8a0ad2845097bb0d05d0a4231

Reply:
Ah I think the error is because a modulation vector hasn't been provided - let me check to see what the expected behaviour is